plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -523,12 +607,73 @@ protected QueryRunner createQueryRunner()
+            regionRowsBuilder.add(new ProducerRecord<>(regionTableName, "key" + row.getField(0), new GenericRecordBuilder(regionSchema)
+                    .set("regionkey", row.getField(0))
+                    .set("name", row.getField(1))
+                    .set("comment", row.getField(2))
+                    .set("updated_at_seconds", initialUpdatedAt.plusMillis(1000).toEpochMilli())
+                    .build()));
+        }
+        kafka.sendMessages(regionRowsBuilder.build().stream(), schemaRegistryAwareProducer(kafka));
+        pinot.createSchema(getClass().getClassLoader().getResourceAsStream("region_schema.json"), regionTableName);
+        pinot.addRealTimeTable(getClass().getClassLoader().getResourceAsStream("region_realtimeSpec.json"), regionTableName);
+
+        String nationTableName = "nation";
+        kafka.createTopicWithConfig(2, 1, nationTableName, false);
+        Schema nationSchema = SchemaBuilder.record(nationTableName).fields()
+                // nationkey BIGINT, name VARCHAR,  VARCHAR, regionkey BIGINT
+                .name("nationkey").type().longType().noDefault()
+                .name("name").type().stringType().noDefault()
+                .name("comment").type().stringType().noDefault()
+                .name("regionkey").type().longType().noDefault()
+                .name("updated_at_seconds").type().longType().noDefault()
+                .endRecord();
+        ImmutableList.Builder<ProducerRecord<String, GenericRecord>> nationRowsBuilder = ImmutableList.builder();
+        MaterializedResult nationRows = queryRunner.execute("SELECT * FROM tpch.tiny.nation");
+        for (MaterializedRow row : nationRows.getMaterializedRows()) {
+            nationRowsBuilder.add(new ProducerRecord<>(nationTableName, "key" + row.getField(0), new GenericRecordBuilder(nationSchema)
+                    .set("nationkey", row.getField(0))
+                    .set("name", row.getField(1))
+                    .set("comment", row.getField(3))
+                    .set("regionkey", row.getField(2))
+                    .set("updated_at_seconds", initialUpdatedAt.plusMillis(1000).toEpochMilli())
+                    .build()));
+        }
+        kafka.sendMessages(nationRowsBuilder.build().stream(), schemaRegistryAwareProducer(kafka));
+        pinot.createSchema(getClass().getClassLoader().getResourceAsStream("nation_schema.json"), nationTableName);
+        pinot.addRealTimeTable(getClass().getClassLoader().getResourceAsStream("nation_realtimeSpec.json"), nationTableName);
+    }
+
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        return switch (connectorBehavior) {
+            case SUPPORTS_CREATE_TABLE_WITH_DATA,
+                    SUPPORTS_CREATE_SCHEMA,
+                    SUPPORTS_CREATE_TABLE,
+                    SUPPORTS_INSERT,
+                    SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS,
```

Comment:
Remove. `SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS` defaults to `SUPPORTS_RENAME_TABLE`.